lib/recipes_bindings.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:get/get_core/src/get_main.dart';
+import 'package:get/get_instance/get_instance.dart';
+import 'presentation/controller/recipe_summary_controller.dart';
+import 'presentation/controller/recipes_controller.dart';
+
+import 'data/datasource/remote/api_service.dart';
+import 'data/repository/recipes_repository.dart';
+import 'domain/usecase/implementations/get_random_recipes_usecase.dart';
+
+class RecipesBinding {
+  void instantiateDependencies() {
```

Comment:
You can instantiate the Dio client here too

Reply:
Done